.github/renovate.json5

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["config:base"],
-  "includePaths": ["package.json", "packages/**", "docusaurus/**", ".github"],
-  "ignoreDeps": ["react", "react-dom", "@stylistic/eslint-plugin-ts", "lerna", "@grafana/scenes"],
+  "includePaths": ["package.json", "packages/**", "docusaurus/**", ".github/**"],
+  "ignoreDeps": ["react", "react-dom", "@stylistic/eslint-plugin-ts", "lerna", "@grafana/scenes", "@libs/output"],
```

Comment:
Why are we ignoring `"@libs/output"` is it because it's an internal lib?

Reply:
yep